src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in

```diff
@@ -8,6 +8,6 @@ Description: @PROJECT_DESCRIPTION@
 URL: @PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
 Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@
-Libs: -L${libdir} -L@Boost_LIBRARY_DIRS@ -lviamapi
+Libs: -Wl,-rpath,${libdir} -L${libdir} -L@Boost_LIBRARY_DIRS@ -lviamapi
```

Comment:
Comparing the `build.ninja` generated in `examples/project/cmake` I saw that `-Wl,-rpath` was getting added to the link line, and I guessed that this might be a way to handle rpath in a pkg-config file, which seems to be confirmed by a big github search to try to compare prior art, see eg https://github.com/search?q=rpath%20path%3A*.pc&type=code

Note also that we only put this in the api `.pc` file because sdk depends on it, and otherwise there will be warnings about duplicate rpaths.

Reply:
Yes, I think it is fine to add this.